php-ts-mode.el

```diff
@@ -245,6 +261,17 @@ see https://www.php.net/manual/language.constants.predefined.php")
      (goto_statement "goto" @php-keyword)
      (yield_expression "from" @php-keyword))
 
+   :language 'php
+   :feature 'built-in
+   :override t
+   `((function_call_expression
+      function: (name) @php-keyword
+      (:match ,(rx-to-string
+                `(seq bol
+                      (or ,@php-ts-mode--built-in-functions)
+                      eol)) @php-keyword))
+     (unset_statement "unset" @php-keyword))
```

Comment:
These keywords are [language constructs](https://www.php.net/manual/en/reserved.keywords.php) that are distinct from [built-in functions](https://www.php.net/manual/en/functions.internal.php) and can be misleading to users. Could you please merge these into `keyword`?

Reply:
@zonuexe  `built-in` feature was removed, rules was merged into `keyword`

Reply:
As we treat `die`, `empty` and `isset` as keywords then we should just move them to `php-ts-mode--keywords`

Reply:
@piotrkwiecinski i tried it first thing, but adding any of those values into `php-ts-mode--keywords` breaks [this rule](https://github.com/emacs-php/php-ts-mode/blob/6c0214eb1a323cfc9cedb5e187c4ed20da763e9b/php-ts-mode.el#L243)
You can check this out by evaluating
```
(treesit-query-validate 'php`(["break"] @php-keyword))
```
Query is valid

```
(treesit-query-validate 'php`(["die"] @php-keyword))
```
Raises node error